src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslation } from 'react-i18next';
+import { useFetch } from '../../../../utils/hooks/useFetch';
+import { Registration } from '../../../../types/registration.types';
+import { SearchApiPath } from '../../../../api/apiPaths';
+import { SearchResponse } from '../../../../types/common.types';
+import { ListSkeleton } from '../../../../components/ListSkeleton';
+import { Divider, List, Typography } from '@mui/material';
+import { CentralImportResultItem } from './CentralImportResultItem';
+import { ResourceFieldNames } from '../../../../types/publicationFieldNames';
+import { createSearchQuery, ExpressionStatement, SearchConfig } from '../../../../utils/searchHelpers';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../../utils/constants';
+
+interface CentralImportDuplicateSearchProps {
+  publication: Registration;
+}
+export const CentralImportDuplicateSearch = ({ publication }: CentralImportDuplicateSearchProps) => {
+  const { t } = useTranslation('basicData');
+
+  const searchConfig: SearchConfig = publication.entityDescription?.reference?.doi
+    ? {
+        properties: [
+          {
+            fieldName: ResourceFieldNames.Doi,
+            value: publication.entityDescription.reference.doi,
+            operator: ExpressionStatement.Contains,
+          },
+        ],
+      }
+    : {};
+
+  const searchQuery = createSearchQuery(searchConfig);
+  const numberOfResults = ROWS_PER_PAGE_OPTIONS[1];
+  const [searchResults, isLoadingSearchResults] = useFetch<SearchResponse<Registration>>({
+    url: `${SearchApiPath.Registrations}?query=${searchQuery}&results=${numberOfResults}`,
+    errorMessage: t('feedback:error.search'),
+  });
+
+  const publications = searchResults?.hits ?? [];
+
+  return (
+    <>
+      {isLoadingSearchResults ? (
+        <ListSkeleton minWidth={100} maxWidth={100} height={100} />
+      ) : (
+        searchResults && (
+          <>
+            <Typography variant="subtitle1">{t('search:hits', { count: searchResults.size })}:</Typography>
+            <Divider />
+            <List>
+              {publications.map((publication) => (
```

Comment:
foreslår å fjerne `const publications` på linje 38, siden det uansett sjekkes for null på liunje 45

```suggestion
              {searchResults.hits.map((publication) => (
```

Reply:
fikset